rosbag2_performance/rosbag2_performance_benchmarking/launch/benchmark_launch.py

```diff
@@ -196,8 +197,20 @@ def _producer_node_exited(event, context):
     # Handle clearing bag files
     if not node_params['preserve_bags']:
         db_files = pathlib.Path.cwd().joinpath(node_params['db_folder']).glob('*.db3')
+        total_size = 0
         for f in db_files:
+            filesize = f.stat().st_size
             f.unlink()
+            # Replace removed storage file with empty file informing about size
+            pathlib.Path.cwd().joinpath(
+                node_params['db_folder'],
+                str(int(filesize/1000000)) + 'MB_' + f.name).touch()
+            total_size += filesize
+
+        # Additional info about total size
+        pathlib.Path.cwd().joinpath(
+            node_params['db_folder'],
+            str(int(total_size/1000000)) + 'MB_TOTAL').touch()
```

Comment:
Am I reading this correctly that you're writing the total size as a file name? It seems much more appropriate to write this information _into_ a file

Reply:
agreed, should be a quick change. @pijaro

Reply:
Indeed, I will write this into `yaml` file.

Reply:
Ok, it is now saved in single `bagfiles_info.yaml` file.